### ISSUE TITLE
Terminal status can not change when using no code upload component.

### ISSUE THREAD

**Issue by yugasun** [serverless/components#807]
# Description

Terminal status can not change when using no code upload component, normally, when run `serverless deploy`, for terminal will output deploy phases, like `Initializing -> Uploading -> Deploying`, but if a component don't need to upload code, the terminal will always display `Initializing`.

For example using [tencent cdn component](https://github.com/serverless-components/tencent-cdn):

![image](https://user-images.githubusercontent.com/8536173/93985044-1e744400-fdb7-11ea-993e-2a236c47d5ca.png)

**Comment by eahefnawy** [comment]
That makes sense though because nothing is uploaded, no?

**Comment by yugasun** [comment]
Yeah, cdn component just help user to config `CDN` and nothing is uploaded.
